book/intro.md

```diff
@@ -4,16 +4,11 @@ Welcome to the AGU24 [GMT](https://docs.generic-mapping-tools.org/6.5)/
 [PyGMT](https://www.pygmt.org/v0.13.0) workshop 🥳! This Jupyter book 📖 contains
 tutorials for making maps 🗺️ and animations 🎦
 
-::::::{grid} 1 1 3 3
-:gutter: 1
-:padding: 1
-
-:::::{grid-item}
-::::{grid} 1 1 1 1
-:gutter: 1
+## 📚 Overview of tutorials
 
+::::{grid}
```

Comment:
> I'd prefer to keep the previous 3x2 layout just so people can see what's in store in one glance rather than having to scroll down. The previous style actually has a flex layout that switches from 3x2 to 1x6 on smaller screens (e.g. mobile devices)

This is a good point. What if they were drop downs so the title is shown and all the details + image is shown on toggle.

Reply:
I agree with @weiji14, I feel it would be good the have all six thumbnails / cards fitting on a screen.